index.js

```diff
@@ -16,6 +16,29 @@ const pkg = require('./package.json')
 const contractName = 'ReplContainer'
 const main = 'Main'
 
+// speacial commands
+const specialCommands = {
```

Comment:
Let's rename this to `globalVars` or `specialGlobals`. I think that will be a little clearer.